environments/_authorized_keys/rnegi.pub

```diff
@@ -1 +1 @@
-ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQCjf0Br3K0yCOeMGdEAVt3R3xccIl9bMKt3/YsLsGwTtLIEpTqxgOF2k5T2LSiIivDcerAqJM4ZGspDADo86aevd5zqoBIyGDWP6WUdCRFrAVgn99QUvk92Y7PcCq9eW5vFO/22TSc6d9Q0s1D4Dq+v+SIRlgHqYoqfNuaBMXjAuaje5no4t+KmAzcXswFtiUTZQYRG6NBvbyjGkioRQYgea5qU3eZayhXnoGN6BGGsI+cz0g0aLbN+hbMbPXBt3f2hmAHUUhEcIwoyq/hq888ug+VMQWEoU7SH/WrArApKjaUV3KxUv8ugy0COsMu5V8xbcn38KR2olHe5SKO6DmjoewYMzcdu6FGWrYizOoBGbFpPXlHNhL5p5weyD46DX5mc/FRvZ2f1Upf0fo2ieBdauW1Y1Q3YD0+0NbS10b/DoRAlZ6WvM6R9KINe2nL4/FU1eFyDx+nwzRbr5Vn6pIhC13pQ0dlFu07N8EleyIgue5N0Fqmo5YdgBOymtxP8UEAAw72TRRECIqt7pKhez8Ce9gpOMKKqTcxEzQxR6zlxbkhw7Oz1h8rlPhWdS1Ybl2yosKJfh4Fp7FeBibJ2aMnzMDr1P3YGtb/AnEB4AO4xhBCWu05idjGUkeF5+oNJc1nWeo7E86NG4kfcUjGCGqzsJEBcaP2nC06cyo7UaH1FSQ== dsi@DSIs-MacBook-Pro.local
\ No newline at end of file
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQC2jkuGMUNIyArvoozx8HLwz3vNEp/Miaasmc1hsguvFw7PweHFgzM8mYxX/saTlJ2zoxz4kJ7SW1FKorGeoTbQBS043cXmMhe3gZVCM33T7bQucAVXjfvo9XMmg6eFWqA5Y+0KS9cld7T8R8bMJCztj5SLLkcmSdM0zTbjpO/buzuSYzYZgJbXFGOqZpZcUdfdkmlMC76Tk+8MgmuGdwKM9tk19WgSvHTx0nS1UHemG6JlZMAK89sbdYIfFfSvrbRhRhvz74jUbp2NVKa/gDuN3RZUcnMD6tANv5SaR6Et9Pfucia6mn0P7kwDZOSEyOcM/WFGIxuOjY8P548rYLSBCffjJPVTtSYlcNPWfLJlba6EFmrw1dZ6x9N5jR+ZX4f3nUMXp4SfMuBYiL6o00GnJs7gbRCU+K9QVVpWsz8WaEM8GlfmdwWMd8jfKQb+KmtQwyHhbbjWhJOlAg1P7sEULbCyLeQ3NoGJsjoWvsVuqPopX3eyAH2djccxK5Mc9IpPaiZmO+m3m/YBJKOT60KiS7Z48rvMsRzpWMvF7MLHIFo5vQkI2ZQrQ+Gj9gaxPtHwWmrc9S/Sk6A/N4mvmmF0nZH+pYvU8kDmh7LHmH5biMc5dfKWxj+TgzZRXfswtA+tUWT2nvbsMeXirQtHZtZ4SFuHeaQupCk70hf4JkqnYw== rnegi@dimagi.com
```

Comment:
does the new line at the end affects the working of this in any way?
Asking because in all other  pub keys does not have a new-line at the end. @rohit-dimagi  can you check please

Reply:
it will work as expected despite having new line.